src/pspm_sf_dcm.m

```diff
@@ -47,19 +51,25 @@
 end
 sts = -1;
 tstart = tic;
+
 %% 2 Check input arguments
-if nargin < 2 || ~isnumeric(sr) || numel(sr) > 1
+% 2.1 set model ---
+try model.scr; catch, warning('Input data is not defined.'); return; end
+try model.sr; catch, warning('Sample rate is not defined.'); return; end
+try model.missing_data; catch, warning('Missing data file is not defined.'); end
```

Comment:
"Missing data index" instead of "file"?

Reply:
I understood it as, if `.missing_data` is not defined, the only reason is `.missing` (missing epoch data file) in `pspm_sf` isn't loaded. What users may need to do is to load the missing epoch file in `pspm_sf`. It should not be possible if users have loaded missing epoch file but `.missing_data` here isn't added.. However, I agree "missing data index" is more explicit, so I have updated it now.

Reply:
Actually, I just realise, there should be no warning if no missing data exists, because this will be the default situation. IMO the warning can be removed entirely.